app/Actions/Project/InviteProjectMember.php

```diff
@@ -34,6 +34,7 @@ public function invite($user, $project, string $email, string $role = null, stri
             'email' => $email,
             'role' => $role,
             'message' => $message,
+            'invited_by' => $user->id,
```

Comment:
Pass the user name and concatenate the name directly to avoid additional DB query